packages/webapp/src/utils/helpers.js

```diff
@@ -235,6 +235,10 @@ const validateIdGetSchemaOptional = {
   }
 }
 const getLandowners = landOwners => {
+  if (!landOwners) {
+    return null
+  }
```

Comment:
Does/could this result in null being rendered?

Reply:
No. It will not render if landowners value is null